pkg/compliance/checks/file_check.go

```diff
@@ -112,28 +116,37 @@ func (c *fileCheck) getAttribute(filePath string, fi os.FileInfo, property strin
 	return "", ErrPropertyNotSupported
 }
 
-func (c *fileCheck) getJSONPathValue(filePath string, jsonquery string) (string, error) {
-	f, err := os.Open(filePath)
-	if err != nil {
+type getter func([]byte, string) (string, error)
```

Comment:
Nit - might make sense to add a comment here even though it isn't exported.

Reply:
done 💯